resources/hiveclients/python/hiveserver1-client.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# imports
+import sys
+import hive_utils
+from hive_service.ttypes import HiveServerException
+
+# get the input parameters
+hiveHost = sys.argv[1]
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c